src/Pact/Types/Term.hs

```diff
@@ -434,12 +468,17 @@ data Module
   , _interfaceCode :: !Code
   , _interfaceMeta :: !Meta
   , _interfaceImports :: [Use]
-  } deriving Eq
-
-instance Show Module where
-  show m = case m of
-    Module{..} -> "(Module " ++ asString' _mName ++ " '" ++ asString' _mKeySet ++ " " ++ show _mHash ++ ")"
-    Interface{..} -> "(Interface " ++ asString' _interfaceName ++ ")"
+  } deriving (Eq, Show)
+
+instance Pretty Module where
+  pretty m = parensSep $ case m of
+    Module{..} ->
+      [ "Module"
```

Comment:
We should conform these kinds of things, with `App` and `Use` being the model, ie sexp-like presentations, and consistent casing (ie lowercase `module` here)

Reply:
Oops, I'll lowercase `module` and `inferface`. Otherwise this already has an sexp presentation so I think it's okay?

Reply:
Yeah, more making the general comment re sexps, agree it's good here.